extensions/css-language-features/server/src/utils/documentContext.ts

```diff
@@ -8,9 +8,10 @@ import { endsWith, startsWith } from '../utils/strings';
 import * as url from 'url';
 import { WorkspaceFolder } from 'vscode-languageserver';
 import URI from 'vscode-uri';
-import { join, dirname } from 'path';
 import { existsSync } from 'fs';
 
+const resolve = require('requireresolveproxy');
```

Comment:
this is due to not having .ts typings for requireresolveproxy - happy to resolve in another way if someone can suggest how